test_data/asm/block_machine_exact_number_of_rows.asm

```diff
@@ -17,15 +17,17 @@ machine Main with min_degree: 32, max_degree: 64 {
     reg A;
 
     ByteBinary byte_binary;
-    // We'll call the binary machine twice and the block size
-    // is 4, so we need exactly 8 rows.
-    Binary binary(byte_binary, 8, 8);
+    // We'll call the binary machine 4 times and the block size
+    // is 4, so we need exactly 16 rows.
+    Binary binary(byte_binary, 16, 16);
```

Comment:
This machine violated the assumption that the number of rows is at least 4x the block size. I don't think it is important to handle these cases.